### PR TITLE
Add Chinese code and service patterns

### PR DESCRIPTION
**Pull request by ye4241**
Try to add Chinese support for code and service patterns.
The Chinese code is same as English for 4-8 digital numbers.
The service pattern is `【([\u4e00-\u9fa5\d\w]+)】` which will support for 【微信支付】 【华为】 【iSlide】 【360】.

### CODE REVIEW

**Comment by itsthisjustin** [comment]
Thanks @ye4241! @alectrocute can you review this? Not sure the best way to test so I guess lets make sure it just doesnt conflict with anything.

File: src/assets/libs/parse-otp-message/index.js [hunk 1]
```diff
@@ -156,7 +156,7 @@ function inferService(message) {
     const match = s.match(pattern)
     const service = match && match[1] && match[1].trim()
 
-    if (service && /\w+/.test(service) && !authWords.has(service)) {
```
Comment (alectrocute) [inline src/assets/libs/parse-otp-message/index.js:159]: Hey @ye4241! What's the purpose of this change?

Comment (ye4241) [reply src/assets/libs/parse-otp-message/index.js:159]: @alectrocute  \w cannot match Chinese, like the service is 【华为】, 华为 is the service，it is not in \w like a-z, A-Z.

Comment (alectrocute) [reply src/assets/libs/parse-otp-message/index.js:159]: @ye4241 Excellent explanation, thank you!

**Review by alectrocute — APPROVED** [review]

**Comment by alectrocute** [comment]
Merged! Thanks for your contribution @ye4241! 🚀
